auth-validation.js

```diff
@@ -0,0 +1,142 @@
+// 비밀번호 관련 기능 모듈 가져오기
+import { pw } from "./toggle-pw.js";
+
+// DOM 요소 참조
+const form = document.querySelector(".js-form");
+const input = document.querySelectorAll(".js-input");
```

Comment:
P2:
`querySelectorAll`은 NodeList를 반환하므로 변수에서도 이를 알 수 있게 네이밍해주세요.

```suggestion
const inputList = document.ㅊ(".js-input");
```